web3.js/README.md

```diff
@@ -23,7 +23,12 @@
 
 This is the Solana Javascript API built on the Solana [JSON RPC API](https://docs.solana.com/apps/jsonrpc-api)
 
-[Latest API Documentation](https://solana-labs.github.io/solana-web3.js/)
+## Documentation and examples
+
+ - [The Solana Cookbook](https://solanacookbook.com/) has extensive task-based documentation using this library.
+ - For more detail on individual functions, see the [latest API Documentation](https://solana-labs.github.io/solana-web3.js/)
+ - [Web3 Examples](https://github.com/solana-labs/solana/tree/master/web3.js/examples)
+ - [Token Program Examples](https://github.com/solana-labs/solana-program-library/tree/master/token/js/examples)
```

Comment:
Are these actually useful anymore? I think we can remove them in favor of the cookbook

Reply:
Up to you - personally I find the cookbook to be a superset of these, but didn't want to remove info in my first PR.

Reply:
I've removed them. :-)

Reply:
Thanks!